config/initializers/opentelemetry.rb

```diff
@@ -0,0 +1,7 @@
+require 'opentelemetry'
```

Comment:
Not sure if it's necessary, but you could add a check to confirm that tracing is enabled prior to running your initializer code `if Rails.application.config.conjur_config.tracing_enabled`